pegasus/helpers/section_api_helpers.rb

```diff
@@ -266,6 +292,10 @@ def self.valid_script_id?(script_id)
     valid_scripts.find {|script| script[:id] == script_id.to_i}
   end
 
+  def self.valid_course_id?(course_id)
```

Comment:
This method appears unused in this PR? Is this intentional? There are plans to use it later?